Testing/Mock/mongo_intro.py

```diff
@@ -0,0 +1,20 @@
+#connect to mongoengine with mongomock
+connect('mongoenginetest', host='mongomock://localhost', alias='testdb')
+conn = get_connection('testdb')
+
+#given code:
+def increase_votes(collection):
+	for document in collection.find():
```

Comment:
I see that this is just something you did for a tutorial, but I do like the idea of keeping the flow of it in the master branch. If you don't want to merge these tests (because they're not super related), maybe keep the frame and put comments in place of code with what needs to be performed in each function? And make them as wordy as possible? That way anyone can copy your template and fill in the meat of it (because you've provided the body). Up to you though whether you want to put all of this stuff into master!